chmgen.d

```diff
@@ -63,6 +64,9 @@ void addKeyword(string keyword, string link, int confidence, bool isAnchor = tru
     if (link.endsWith("/"))
         link ~= "index.html";
 
+    if (prerelease && link.startsWith("phobos/"))
```

Comment:
nit: could be `link.skipOver("phobos/")` - then you would save the `link[7 .. $]` below.